command-parser/README.md

```diff
@@ -4,6 +4,11 @@
 
 [![codecov badge][]][codecov link] [![discord badge][]][discord link] [![github badge][]][github link] [![license badge][]][license link] ![rust badge]
 
+**This crate has been deprecated. Please use [Discord interactions] via the
+[`twilight-gateway`] or [`twilight-http`] crates.**
+
+**This crate has been deprecated. Please see the above notice.**
```

Comment:
would this be better as a header?

Reply:
I've gone ahead and removed a line actually. This should be very apparent by the documentation:

<img width="1020" alt="Screen Shot 2021-12-26 at 09 36 14" src="https://user-images.githubusercontent.com/57759500/147411374-925d2588-40b0-4486-9c14-ce969b8d33bb.png">